source/simulator/parameters.cc

```diff
@@ -550,45 +520,40 @@ namespace aspect
-                         "Choose none, one or several from "
-                         "\n\n"
-                         "\\begin{itemize} \\item net rotation \\item angular momentum \\item net translation "
-                         "\\item linear momentum \\item net x translation \\item net y translation "
-                         "\\item net z translation \\item linear x momentum \\item linear y momentum "
-                         "\\item linear z momentum \\end{itemize}"
-                         "\n\n"
-                         "These are a selection of operations to remove certain parts of the nullspace from "
-                         "the velocity after solving. For some geometries and certain boundary conditions "
-                         "the velocity field is not uniquely determined but contains free translations "
-                         "and/or rotations. Depending on what you specify here, these non-determined "
-                         "modes will be removed from the velocity field at the end of the Stokes solve step.\n"
-                         "\n\n"
-                         "The ``angular momentum'' option removes a rotation such that the net angular momentum "
-                         "is zero. The ``linear * momentum'' options remove translations such that the net "
-                         "momentum in the relevant direction is zero.  The ``net rotation'' option removes the "
-                         "net rotation of the domain, and the ``net * translation'' options remove the "
-                         "net translations in the relevant directions.  For most problems there should not be a "
-                         "significant difference between the momentum and rotation/translation versions of "
-                         "nullspace removal, although the momentum versions are more physically motivated. "
-                         "They are equivalent for constant density simulations, and approximately equivalent "
-                         "when the density variations are small."
-                         "\n\n"
-                         "Note that while more than one operation can be selected it only makes sense to "
-                         "pick one rotational and one translational operation.");
-      prm.declare_entry ("Enable additional Stokes RHS", "false",
-                         Patterns::Bool (),
-                         "Whether to ask the material model for additional terms for the right-hand side "
-                         "of the Stokes equation. This feature is likely only used when implementing force "
-                         "vectors for manufactured solution problems and requires filling additional outputs "
-                         "of type AdditionalMaterialOutputsStokesRHS.");
-
     }
-    prm.leave_subsection ();
+    prm.leave_subsection();
+
+
+    prm.declare_entry ("Remove nullspace", "",
```

Comment:
Consider adding a separate subsection from the nullspace removal (`Nullspace removal`).